python_modules/dagster/dagster/_core/definitions/job_definition.py

```diff
@@ -489,13 +491,20 @@ def _get_job_def_for_op_selection(
         try:
             sub_graph = get_subselected_graph_definition(self.graph, resolved_op_selection_dict)
 
+            # if explicit config was passed the config_mapping that resolves the defaults implicitly is
+            # very unlikely to work. The preset will still present the default config in dagit.
+            if self._explicit_config:
+                config_arg = None
+            else:
+                config_arg = self.config_mapping or self.partitioned_config
```

Comment:
trying to think of cases where people would actually get by with the existing behavior
* no dependencies in the graph such that subsetting doesn't change the schema
* ???